docs/reuse/theme.md

```diff
@@ -0,0 +1,6 @@
+You can view the user interface in a dark theme or a light theme:
```

Comment:
maybe we could title this file something like `dark-light-theme.md` to make it easier to find down the road

Reply:
Excellent idea, @kimsauce! Done.